### PR TITLE
Update Node version to current LTS release

### PR DESCRIPTION
**Pull request by mattlord**
## Description

At least on main/v20 we should use the current LTS version: https://nodejs.org/en/about/previous-releases

We also update the dependencies using the latest min node version of 20.12.0 using `cd web/vtadmin && npm update`.

## Checklist

-   [x] "Backport to:" labels have been added if this change should be back-ported to release branches
-   [x] If this change is to be back-ported to previous releases, a justification is included in the PR description
-   [x] Tests were added or are not required
-   [x] Did the new or modified tests pass consistently locally and on CI?
-   [x] Documentation was added or is not required

### CODE REVIEW

**Review by notfelineit — APPROVED** [review]
Thank you!

**Review by frouioui — COMMENTED** [review]
We should change the following files too:
- `web/vtadmin/build.sh` (used by the Makefile when doing `make build` and by Docker)
- `web/vtadmin/README.md`

**Comment by mattlord** [comment]
> We should change the following files too:
> 
> * `web/vtadmin/build.sh` (used by the Makefile when doing `make build` and by Docker)
> * `web/vtadmin/README.md`

Good catch! I've done that now. I've also added a note about this to the release summary doc. /cc @notfelineit

**Review by frouioui — APPROVED** [review]
VTAdmin tests are failing, but otherwise the content of the PR looks good to me. Thank you for doing this @mattlord 🚀

File: changelog/20.0/20.0.0/summary.md [hunk 1]
```diff
@@ -248,3 +250,10 @@ VTTablet exposes two new counter stats:
 ### <a id="sighup-reload-of-grpc-client-auth-creds"/>`SIGHUP` reload of gRPC client static auth creds
 
 The internal gRPC client now caches the static auth credentials and supports reloading via the `SIGHUP` signal. Previous to v20 the credentials were not cached. They were re-loaded from disk on every use.
+
+### <a id="vtadmin"/>VTAdmin
+
+#### <a id="updated-node"/>vtadmin-web updated to node v20.12.2 (LTS)
+
+Building `vtadmin-web` now requires node >= v20.12.0 (LTS). Breaking changes from v18 to v20 can be found at https://nodejs.org/en/blog/release/v20.12.0 -- with no known issues that apply to VTAdmin.
```
Comment (notfelineit) [inline changelog/20.0/20.0.0/summary.md:258]: This is great, thank you

**Review by notfelineit — APPROVED** [review]

**Pull request merged**